Documentation/proposals/202407-remote-write.md

```diff
@@ -0,0 +1,121 @@
+# RemoteWrite CRD
+
+- Owners:
+  - [@superbrothers](https://github.com/superbrothers)
+- Related Tickets:
+  - [#6508](https://github.com/prometheus-operator/prometheus-operator/issues/6508)
+- Other docs:
+  - n/a
+
+## TL;DR
+
+This design doc proposes RemoteWrite CRD, which enables cluster admins to delegate the ability to configure Prometheus remote_rewrite configuration to application developers/operators.
+
+## Why
+
+The Prometheus remote_write configuration is defined in the Prometheus CRD. Currently, the configuration data generation is the responsibility of cluster admins.
+
+## Goals
+
+The main goal is for application developers/operators to configure themselves how the remote endpoint sends metrics.
+
+This means exposing a new CRD to configure Prometheus remote_write configuration.
+
+## Non-goals
+
+We do not cover "RemoteRead" CRD here. We can still implement it later if needed.
+
+## How
+
+### RemoteWrite CRD
+
+The RemoteWrite CRD represents one of the Prometheus remote_write configuration scoped to the resource’s namespace.
+
+```yaml
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: RemoteWrite
+metadata:
+  name: example
+  namespace: default
+spec:
+  url: "https://aps-workspaces.us-west-2.amazonaws.com/workspaces/<workspace id>/api/v1/remote_write"
+  sigv4:
+    region: us-west-2
+    accessKey:
```

Comment:
It might be good to mention that secret/configmap refs will be resolved in the RemoteWrite namespace.

Reply:
```suggestion
    # Secret object references will be resolved in the namespace of this remoteWrite object.
    # In this manifest it is the "default" namespace.
    accessKey:
```